### PR TITLE
{vis}[GCCcore/6.4.0] libav v11.10

### PR DESCRIPTION
**Pull request by JackPerdue**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/l/libav/libav-11.10-GCCcore-6.4.0.eb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+description = """
+ Libav is a friendly and community-driven effort to provide its users with
+ a set of portable, functional and high-performance libraries for dealing
+ with multimedia formats of all sorts.
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '6.4.0'}
+toolchainopts = {'pic': True}
+
+# https://libav.org/releases/libav-11.9.tar.gz
```
Comment (boegel) [inline easybuild/easyconfigs/l/libav/libav-11.10-GCCcore-6.4.0.eb]: @JackPerdue please remove this outdated comment

**Review by boegel — DISMISSED** [review]

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2724.swalot.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/b2785ec1dee4c8bea953a14b6cf453f5 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2121.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/d86a4fa7348ec61dd088f0580ce2d20d for a full test report.

**Comment by migueldiascosta** [comment]
@JackPerdue, this PR is being closed for the following reason(s): closing and reopening to trigger tests.

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 0 out of 0 (1 easyconfigs in this PR)
b-an04.hpc2n.umu.se - Linux ubuntu 16.04, x86_64, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 2.7.12
See https://gist.github.com/5e42de897b794009f04a16039467669b for a full test report.

File: easybuild/easyconfigs/l/libav/libav-11.10-GCCcore-6.4.0.eb (hunk 1 above)
Comment (akesandgren) [reply easybuild/easyconfigs/l/libav/libav-11.10-GCCcore-6.4.0.eb]: Done

**Review by akesandgren — APPROVED** [review]
LGTM

**Comment by akesandgren** [comment]
Going in, thanks @JackPerdue!

**Pull request merged**